code/modules/vehicles/unmanned/unmanned_vehicle.dm

```diff
@@ -10,10 +10,12 @@
 	light_power = 3
 	light_system = MOVABLE_LIGHT
 	move_delay = 2.5	//set this to limit the speed of the vehicle
-	max_integrity = 300
+	max_integrity = 150
 	hud_possible = list(MACHINE_HEALTH_HUD, MACHINE_AMMO_HUD)
 	flags_atom = BUMP_ATTACKABLE
-	soft_armor = list("melee" = 25, "bullet" = 50, "laser" = 50, "energy" = 100, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 25, "acid" = 25)
+	soft_armor = list("melee" = 25, "bullet" = 85, "laser" = 50, "energy" = 100, "bomb" = 50, "bio" = 100, "rad" = 100, "fire" = 25, "acid" = 25)
+	///allows it to be put in crate
+	density = FALSE
```

Comment:
You are aware this makes it possible to walk through, doesnt bump and (iirc) gives it ranged attack immunity?

Reply:
it doesnt give it ranged attack immunity but yea it does make it possible to walk through